doc/rest-api.md

```diff
@@ -1818,3 +1818,278 @@ Input (none at present):
     }
 
 HTTP code for this should be 202 (Accepted).
+
+## /1.0/storage-pools
+### GET
+ * Description: list of storage pools
+ * Introduced: with API extension "storage"
+ * Authentication: trusted
+ * Operation: sync
+ * Return: list of storage pools that are currently defined on the host
+
+
+    [
+        {
+            "name": "pool4",
+            "driver": "btrfs",
+            "used_by": [],
+            "config": {
+                "size": "61205515468",
+                "source": "/home/chb/mnt/lxd_clean/disks/pool4.img",
+                "volume.size": "0"
+            }
+        },
+        {
+            "name": "test",
+            "driver": "lvm",
+            "used_by": [
+                "/1.0/containers/alp1"
+                "/1.0/containers/alp1/snapshots/snap0",
+                "/1.0/containers/alp1/snapshots/snap1",
+                "/1.0/containers/alp1/snapshots/snap2",
+                "/1.0/containers/alp2",
+                "/1.0/containers/alp3",
+                "/1.0/images/cedce20b5b236f1071134beba7a5fd2aa923fda49eea4c66454dd559a5d6e906"
+            ]
+        }
+    ]
+
+### POST
+ * Description: create a new storage pool
+ * Introduced: with API extension "storage"
+ * Authentication: trusted
+ * Operation: sync
+ * Return: standard return value or standard error
+
+Input:
+
+    {
+        "config": {
+            "size": "10GB"
+        },
+        "driver": "zfs",
+        "name": "pool1"
+    }
+
+## /1.0/storage-pools/<name>
+### GET
+ * Description: information about a storage pool
+ * Introduced: with API extension "storage"
+ * Authentication: trusted
+ * Operation: sync
+ * Return: dict representing a storage pool
+
+    {
+        "/1.0/containers/alp1",
+        "/1.0/containers/alp1/snapshots/snap0",
+        "/1.0/containers/alp1/snapshots/snap1",
+        "/1.0/containers/alp1/snapshots/snap2",
+        "/1.0/containers/alp2",
+        "/1.0/containers/alp3",
+        "/1.0/images/cedce20b5b236f1071134beba7a5fd2aa923fda49eea4c66454dd559a5d6e906",
+    }
```

Comment:
Hm, is the comment misplaced? Man, this Github inline commenting is driving me nuts. :laughing: